src/app/routes/article/getInitialData/index.test.ts

```diff
@@ -16,6 +16,7 @@ const bffArticleJson = {
       content: {},
       metadata: {
         allowAdvertising: true,
+        consumableAsSFV: true,
```

Comment:
I just wanted to check - is this returned from the article fabl-module?

Reply:
It is indeed, sample response from FABL: 
![Screenshot 2023-03-08 at 16 56 19](https://user-images.githubusercontent.com/32307964/223778306-6ff9fe2b-05f5-4b6e-b3d4-4f8375945f5a.png)